src/frontend/overlay/src/components/organisms/widgets/Locator.jsx

```diff
@@ -65,7 +66,7 @@ export const Locator = ({ widgetData, transitionState }) => {
             <rect x="0" y="0" className="backdrop" height="100%" width="100%"/>
         </svg>
         {circles.map((circle, index) => {
-            let left = circle.x - 540 / 2;
+            let left = circle.x - 343 / 2;
```

Comment:
I have no idea, on what does 540 and 343 mean,s but based on how many times it used, it should be a constant.